doc/whats_new/v0.20.rst

```diff
@@ -233,6 +233,9 @@ Metrics
   False and both inputs are sparse, will return a sparse matrix.
   :issue:`10999` by :user:`Taylor G Smith <tgsmith61591>`.
 
+- :func:`metrics.log_loss` now supports multi-label log loss when input is 2D
```

Comment:
I think you need to list this as a bug fix and explain clearly that behaviour has been corrected. Multilabel was measured by this function before, but incorrectly